Core/Rewriters/MockInstantiationRewriter.cs

```diff
@@ -38,35 +38,69 @@ public MockInstantiationRewriter (IFormatter formatter)
         throw new InvalidOperationException ("Rhino.Mocks cannot be found.");
       }
 
+      var trackedNodes = node.TrackNodes (node.DescendantNodesAndSelf(), CompilationId);
```

Comment:
You're only using `GetOriginalNode` to resolve `baseCallNode`, which results in `node` in this scope. Am I missing something?

Reply:
> You're only using `GetOriginalNode` to resolve `baseCallNode`, which results in `node` in this scope. Am I missing something?

I need to track all descendant invocationexpressions and the node itself, because after the first `TrackNodes` every node in the tree is invalid and i would not be able to get the original node in a base call

Reply:
Ah right, I didn't think that through. That would mean you only need to track the node itself and descendant invocation expressions, right?

Reply:
yes